egs/wsj/asr2/conf/preprocess.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need this file?

Reply:
No. I will remove this. I just wanna confirmed our JSON is compatible to YAML format.